.bazelrc

```diff
@@ -219,13 +219,16 @@ build:mkl_aarch64_threadpool -c opt
 build:cuda --repo_env TF_NEED_CUDA=1
 build:cuda --crosstool_top=@local_config_cuda//crosstool:toolchain
 build:cuda --@local_config_cuda//:enable_cuda
+# Default CUDA and CUDNN versions.
+build:cuda --repo_env=HERMETIC_CUDA_VERSION="12.3.2"
+build:cuda --repo_env=HERMETIC_CUDNN_VERSION="8.9.7.29"
+# This flag is needed to include hermetic CUDA libraries for bazel tests.
+test:cuda --@local_config_cuda//cuda:include_hermetic_cuda_libs=true
 
 # CUDA: This config refers to building CUDA op kernels with clang.
 build:cuda_clang --config=cuda
-# Enable TensorRT optimizations https://developer.nvidia.com/tensorrt
```

Comment:
To enable TensorRT optimization, follow these steps:
    1. Install TensorRT: Ensure you have TensorRT installed. You can use a container, a Debian file, or a standalone pip wheel file.
    2. Model Conversion: Convert your model to a format compatible with TensorRT, such as ONNX.
    3. Configuration: Add the necessary optimization configurations to your model's configuration file.
    4. Deployment: Deploy your optimized model using the TensorRT runtime.
For detailed instructions, you can refer to the [NVIDIA TensorRT Quick Start Guide](https://docs.nvidia.com/deeplearning/tensorrt/quick-start-guide/index.html) and the [Triton Inference Server documentation](https://docs.nvidia.com/deeplearning/triton-inference-server/archives/triton_inference_server_1140/user-guide/docs/optimization.html)[1](https://docs.nvidia.com/deeplearning/tensorrt/quick-start-guide/index.html)[2](https://docs.nvidia.com/deeplearning/triton-inference-server/archives/triton_inference_server_1140/user-guide/docs/optimization.html).

1. [Quick Start Guide :: NVIDIA Deep Learning TensorRT Documentation](https://docs.nvidia.com/deeplearning/tensorrt/quick-start-guide/index.html)
2. [Optimization — NVIDIA Triton Inference Server 2.0.0 documentation](https://docs.nvidia.com/deeplearning/triton-inference-server/archives/triton_inference_server_1140/user-guide/docs/optimization.html)
3. [Graph optimizations | onnxruntime](https://onnxruntime.ai/docs/performance/model-optimizations/graph-optimizations.html)
4. [Best Practices for Tuning the Performance of TensorRT-LLM](https://nvidia.github.io/TensorRT-LLM/performance/perf-best-practices.html)

Reply:
We are planning to get rid of TensorRT support in the new TF release.